spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -174,26 +175,22 @@ protected void onInit() {
 
 	@SuppressWarnings("unchecked")
 	private Runnable createPoller() throws Exception {
-		List<Advice> receiveOnlyAdviceChain = new ArrayList<Advice>();
+		List<Advice> receiveOnlyAdviceChain = new ArrayList<>();
```

Comment:
`... = null`

Directly assign result of `collect()` and add else clause to set as `ArrayList<>()`.